shared/base.js

```diff
@@ -12,16 +12,16 @@ function createContextMenu() {
   });
 }
 
-function enableOrDisable(tabId, changeInfo, tab) {
-  function isReportableURL(url) {
-    if (!url) {
-      return false;
-    }
-
-    let protocol = new URL(url).protocol;
-    return ["http:", "https:"].includes(protocol);
+export function isReportableURL(url) {
```

Comment:
Rather than export from here, it would be better to make this function it's own module, which is imported *here* (since it's used inside `enableOrDisable`).

Reply:
I'll make the necessary changes!

Reply:
By module, you mean a separate file?

Reply:
@OrionStar25 yep, it will be its own file. Check out the links at https://github.com/webcompat/webcompat-reporter-extensions/pull/111#issuecomment-372542338